index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Hello Microverse</title>
```

Comment:
- [ ] [OPTIONAL] Add a link tag for your styles file. This will bring about the styles you created to the page

Reply:
Thanks again..